client/blocks/product-purchase-features-list/sell-online-paypal.jsx

```diff
@@ -10,25 +10,46 @@ import { localize } from 'i18n-calypso';
  */
 import PurchaseDetail from 'components/purchase-detail';
 import { localizeUrl } from 'lib/i18n-utils';
+import { isEnabled } from 'config';
 
 /**
  * Image dependencies
  */
 import paymentsImage from 'assets/images/illustrations/payments.svg';
 
 export default localize( ( { isJetpack, translate } ) => {
+	let supportDocLink;
+	if ( isEnabled( 'earn/rename-payment-blocks' ) ) {
+		supportDocLink = localizeUrl(
+			isJetpack
+				? 'https://jetpack.com/support/pay-with-paypal/'
+				: 'https://wordpress.com/support/pay-with-paypal/'
+		);
+	} else {
+		supportDocLink = localizeUrl(
+			isJetpack
+				? 'https://jetpack.com/support/simple-payment-button/'
+				: 'https://wordpress.com/support/simple-payments/'
+		);
+	}
 	return (
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
-				buttonText={ translate( 'Collect payments' ) }
-				description={ translate(
-					'Add a payment button to any post or page to collect PayPal payments for physical products, digital goods, services, or donations.'
-				) }
-				href={ localizeUrl(
-					isJetpack
-						? 'https://jetpack.com/support/simple-payment-button/'
-						: 'https://wordpress.com/support/simple-payments/'
-				) }
+				buttonText={
+					isEnabled( 'earn/rename-payment-blocks' )
+						? translate( 'Collect PayPal payments' )
+						: translate( 'Collect payments' )
+				}
+				description={
+					isEnabled( 'earn/rename-payment-blocks' )
+						? translate(
+								'Add a button to any post or page to collect PayPal payments for physical products, digital goods, services, or donations.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Add a payment button to any post or page to collect PayPal payments for physical products, digital goods, services, or donations.' )` *ES Score: 9*